tests/test_e2e.py

```diff
@@ -0,0 +1,87 @@
+import pytest
+
+from neptune_prophet.impl import (
+    create_forecast_plots,
+    create_residual_diagnostics_plots,
+    create_summary,
+    get_forecast_components,
+    get_model_config,
+    get_serialized_model,
+)
+
+try:
+    # neptune-client=0.9.0+ package structure
+    import neptune.new as neptune
+except ImportError:
+    # neptune-client>=1.0.0 package structure
+    import neptune
+
+
+@pytest.mark.parametrize("log_interactive", [False, True])
+def test_e2e(dataset, model, log_interactive):
+
+    if log_interactive:
+        try:
+            import plotly  # pylint: disable=import-outside-toplevel, unused-import
+        except ModuleNotFoundError:
+            pytest.skip("plotly is needed for log_interactive to work")
+
+    run = neptune.init(
+        project="common/fbprophet-integration",
```

Comment:
These values will be provided by Github. Leave it as `neptune.init()`

Reply:
But the tests won't work locally in such a case?

Reply:
You're still able to set it with env variables. `common` shouldn't be for testing as there is no cleaning up logic.

Reply:
Done.